nav2_docking/opennav_docking_core/include/opennav_docking_core/charging_dock.hpp

```diff
@@ -123,6 +129,13 @@ class ChargingDock
    */
   virtual bool isCharger() {return true;}
 
+  /**
+   * @brief Indicates the direction of the dock. This is used to determine if the
+   * robot should drive forwards or backwards onto the dock.
+   * @return DockDirection The direction of the dock
+   */
+  virtual DockDirection getDockDirection() {return DockDirection::FORWARD;}
```

Comment:
This could simply be a boolean for forward/back. Do you see other cases we'd want to add?